lib/Braintree/WebhookTestingGateway.php

```diff
@@ -88,6 +88,15 @@ private static function _sampleXml($kind, $id, $sourceMerchantId)
             case WebhookNotification::SUBSCRIPTION_CHARGED_UNSUCCESSFULLY:
                 $subjectXml = self::_subscriptionChargedUnsuccessfullySampleXml($id);
                 break;
+            case WebhookNotification::SUBSCRIPTION_EXPIRED:
+                $subjectXml = self::_subscriptionExpired($id);
+                break;
+            case WebhookNotification::SUBSCRIPTION_CANCELED:
+                $subjectXml = self::_subscriptionCanceled($id);
+                break;
+            case WebhookNotification::SUBSCRIPTION_WENT_PAST_DUE:
+                $subjectXml = self::_subscriptionWentPastDue($id);
+                break;
```

Comment:
Don't these need to be updated to have the correct method names now that they have had `SampleXml` to them?

Reply:
I`m sorry. I fixed it.